README.rst

```diff
@@ -16,7 +16,7 @@ submit a pull request. However, **do not fork the master branch** - fork the
 .. _lektor branch: https://github.com/pybee/pybee.github.io/tree/lektor
 
 If you want to test out a change before you submit it, download and install
-`Lektor`_ (note that Lektor is still using Python 2.7):
+`Lektor`_ (note: Lektor can use Python 2.7 or Python 3.x):
```

Comment:
We shouldn't recommend using Python 3 just yet, because although it kind of works in Python 3, there are some things that don't quite work properly.
To avoid giving wrong advice, that text would be better as: `(note: currently it's better to use Lektor with Python 2.7)`.